src/go.mod

```diff
@@ -57,6 +57,7 @@ require (
 	github.com/opencontainers/go-digest v1.0.0-rc1
 	github.com/opencontainers/image-spec v1.0.1
 	github.com/opentracing/opentracing-go v1.1.0 // indirect
+	github.com/pkg/errors v0.9.1
```

Comment:
Wrong ?

Reply:
It's added automatically when run go build/test/etc.
It is used by https://github.com/goharbor/harbor/blob/master/src/pkg/p2p/preheat/provider/preheat_image.go#L21